### PR TITLE
SFD-4032 Dont authenticate certain endpoints and swagger update

### PR DESCRIPTION
**Pull request by ArifNhs**
SFD-4032 Dont authenticate certain endpoints and swagger update

### CODE REVIEW

File: application/app/src/main/java/uk/nhs/digital/uec/api/authentication/config/MockAuthConfig.java [hunk 1]
```diff
@@ -16,18 +19,45 @@
 @Configuration
 public class MockAuthConfig extends WebSecurityConfigurerAdapter {
 
+  public static final String LOGIN_URL = "/authentication/login";
+  public static final String HEALTH_CHECK_READINESS_URL = "/actuator/health/readiness";
+  public static final String HEALTH_CHECK_LIVENESS_URL = "/actuator/health/liveness";
+  public static final String WELCOME_URL = "/dosapi/dosservices/v0.0.1/home";
```
Comment (jope1) [inline application/app/src/main/java/uk/nhs/digital/uec/api/authentication/config/MockAuthConfig.java:25]: Please change the welcome home url to /api/home

File: application/app/src/main/resources/application.yaml [hunk 2]
```diff
@@ -25,8 +25,8 @@ api:
     service_search: ${API_SERVICE_SEARCH_ENDPOINT}
   cors:
     allowedorigins: ${ALLOWED_ORIGINS}
-  version: 1.0
-  title: NHS Fuzzy Search Application Interface
```
Comment (jope1) [inline application/app/src/main/resources/application.yaml:29]: title: Service Fuzzy Search API

File: application/app/src/main/resources/application.yaml [hunk 3]
```diff
@@ -25,8 +25,8 @@ api:
...
+  version: 1.0.0
+  title: NHS Fuzzy Search API
   description: NHS Fuzzy search APIs provide NHS Services information to the users
```
Comment (jope1) [inline application/app/src/main/resources/application.yaml:30]: Description: Provides service information given a set of fuzzy search criteria.

**Review by jope1 — CHANGES_REQUESTED** [review]

File: application/app/src/main/resources/application.yaml (hunk 3 above)
Comment (ArifNhs) [reply application/app/src/main/resources/application.yaml:30]: Thanks Jon, let's discuss more details on the criteria in swagger description.

**Review by jope1 — APPROVED** [review]

**Pull request merged**